server/db/models/order-details.js

```diff
@@ -0,0 +1,25 @@
+// define OrderDetails model here
+const Sequelize = require('sequelize');
+const { db } = require('./../database.js');
+
+const { UUID, UUIDV4, DECIMAL, INTEGER } = Sequelize;
+
+const OrderDetails = db.define('orderDetails', {
+    orderDetailsId: {
+        //how to define the orderDetailsId as concat(orderId, productId)?
+        primaryKey: true,
+        type: UUID,
+    },
+    productQuantity: {
+        type: INTEGER,
+        allowNull: false,
+    },
+    productCost: {
+        //how to make product cost as the product
```

Comment:
This should be something we can take care of elsewhere. This looks correct to me.